### DISCUSSION TITLE
Not working the devtools for react project

### DISCUSSION THREAD

**Discussion by Bhargav1224** [reduxjs/redux-devtools#1041] [General]
I am using react app along with redux devtools , in devtools everything  is
 showing undefined

**Comment by Methuselah96** [comment]
Can you provide an example project for me to investigate?

**Reply by VHMartinezHernandez** [reply]
> ¿Puede proporcionarme un proyecto de ejemplo para investigar?

Me esta pasando lo mismo y no puedo usar la herramienta, me pueden ayudar?

**Comment by ghost** [comment]
i am also facing the same issue. Is there any resolution to this ?
